sdk/tools/mkisofs/CMakeLists.txt

```diff
@@ -3,7 +3,11 @@ add_definitions(
     -DDUPLICATES_ONCE
     -DINS_BASE="\\\".\\\""
     -DSCHILY_BUILD
-    -DSORTING)
+    -DSORTING
+    -DHAVE_ENVIRON_DEF
+    -DHAVE_RENAME
+    -DHAVE_STRNLEN
+    -D_CRT_NONSTDC_NO_DEPRECATE)
```

Comment:
These defines are meant to be added to sdk/tools/mkisofs/reactos/xconfig.h and possibly guarded with `#if defined(_MSC_VER)` or similar checks.
This file handles all different host OS and compiler combinations as nice as possible (Windows, Linux, Mac OS, GCC, MSVC, Clang).

Reply:
Fixed